middleware/http/routeralias/routeralias_middleware.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package routeralias
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/dapr/components-contrib/middleware"
+	"github.com/dapr/kit/logger"
+
+	"github.com/julienschmidt/httprouter"
+	"github.com/valyala/fasthttp"
+)
+
+// Middleware is an routeralias middleware.
+type Middleware struct {
+	logger logger.Logger
+	router *httprouter.Router
+}
+
+// NewMiddleware returns a new routerchecker middleware.
+func NewMiddleware(logger logger.Logger) middleware.Middleware {
+	return &Middleware{logger: logger}
+}
+
+// GetHandler retruns the HTTP handler provided by the middleware.
+func (m *Middleware) GetHandler(metadata middleware.Metadata) (
+	func(h fasthttp.RequestHandler) fasthttp.RequestHandler, error,
+) {
+	if err := m.getNativeMetadata(metadata); err != nil {
+		return nil, err
+	}
+	return func(h fasthttp.RequestHandler) fasthttp.RequestHandler {
+		return func(ctx *fasthttp.RequestCtx) {
+			uri := ctx.Request.URI()
+
+			handle, params, _ := m.router.Lookup("", string(uri.Path()))
```

Comment:
Please handle error.

Reply:
The last parameter returned `_` is bool type. 

```golang
// Lookup allows the manual lookup of a method + path combo.
// This is e.g. useful to build a framework around this router.
// If the path was found, it returns the handle function and the path parameter
// values. Otherwise the third return value indicates whether a redirection to
// the same path with an extra / without the trailing slash sh bool e performed.
func (r *Router) Lookup(method, path string) (Handle, Params, bool) {
        if root := r.trees[method]; root != nil {
                return root.getValue(path)
        }
        return nil, nil, false
}
```